modules/admin/proc-install-operator.adoc

```diff
@@ -19,23 +23,43 @@ Containers are available for the following CPU architectures:
 * You have configured the appropriate roles and permissions within your project to create an application. For more information, see the link:https://docs.openshift.com/container-platform/{ocp-version}/applications/index.html[Red Hat OpenShift documentation on Building applications].
 
-.Procedure
+[NOTE]
+====
+For enhanced security, deploy the {product} Operator in a dedicated default namespace such as `rhdh-operator`. The cluster administrator can restrict other users' access to the Operator resources through role bindings or cluster role bindings.
+====
 
-. In the *Administrator* perspective of the {ocp-short} web console, navigate to *Operators* > *OperatorHub*.
+.Procedure
 
+. In the *Administrator* perspective of the {ocp-short} web console, navigate to *Operators* -> *OperatorHub*.
 . In the *Filter by keyword* box, enter {product-short} and click the *{product}* Operator card.
-
 . On the *{product} Operator* page, click *Install*.
 
-. On the *Install Operator* page, click *Install*.
+. On the *Install Operator* page, use the *Update channel* drop-down menu to choose an update channel:
+** The *fast* channel provides y-stream (x.y) and z-stream (x.y.z) updates, for example, updating from version 1.0 to 1.1, or from 1.1.0 to 1.1.1.
++
+[IMPORTANT]
+====
+The `fast` channel includes all updates, any may introduce breaking changes in your {product} deployment.
+====
 
-.Verification
-To view the installed {product} Operator, click *View operator*.
+** The *fast-1.1* channel only provides z-stream updates, for example, updating from version 1.1.1 to 1.1.2. If you want to update the {product} y-version in the future, for example, updating from 1.0 to 1.1, you must switch to the *fast* channel manually.
 
+. On the *Install Operator* page, choose the *Update approval* strategy for the Operator:
+** If you choose the *Automatic* option, the Operator is updated without requiring manual confirmation.
+** If you choose the *Manual* option, when a new update is released in the update channel, a notification opens, and the update must be manually approved by an administrator before installation can begin.
++
 [NOTE]
-For enhanced security, deploy the {product} Operator in a dedicated default namespace such as `rhdh-operator`. The cluster administrator can restrict other users' access to the operator resources through role bindings or cluster role bindings. You can choose to deploy the operator in the `openshift-operators` namespace instead, however, the {product} operator shares the namespace with other operators, and therefore any users who can create workloads in that namespace can get their privileges escalated from all operators' service accounts.
+====
+If you are installing in an airgapped or proxied environment where it is required that you copy container images into your cluster manually, use this update strategy to ensure seamless upgrades can occur after the airgap preparation has been completed.
```

Comment:
for context we could link to the section on airgapped operator install, eg. to https://access.redhat.com/documentation/en-us/red_hat_developer_hub/1.1/html-single/administration_guide_for_red_hat_developer_hub/index#proc-install-rhdh-airgapped-environment-ocp-operator_admin-rhdh

Reply:
I'm actually not sure we should mention this at all looking at the docs - this procedure is about using the OCP web console, whereas https://access.redhat.com/documentation/en-us/red_hat_developer_hub/1.1/html-single/administration_guide_for_red_hat_developer_hub/index#proc-install-rhdh-airgapped-environment-ocp-operator_admin-rhdh uses only the CLI, so I'm not sure how this would fit in. I would say maybe instead to remove this note so that we're not having scope creep on this PR, and if we need to talk about channels for airgapped installations, open a new Jira to update the docs for that?